core/src/test/scala/com/pingcap/tispark/convert/ToTimestampSuite.scala

```diff
@@ -87,15 +86,16 @@ class ToTimestampSuite extends BaseDataSourceTest("test_data_type_convert_to_tim
         // insert rows
         writeFunc(List(row1, row2), schema, None)
         compareTiDBSelectWithJDBC(Seq(readRow1, readRow2), readSchema)
+      case (writeFunc, "jdbcWrite") =>
+      // ignored because spark jdbc write does not use local time zone
     }
   }
 
-  // TODO: to support
-  ignore("Test Convert from java.sql.Date to TIMESTAMP") {
+  test("Test Convert from java.sql.Date to TIMESTAMP") {
     // success
     // java.sql.Date -> TIMESTAMP
     compareTiDBWriteWithJDBC {
-      case (writeFunc, _) =>
+      case (writeFunc, "tidbWrite") =>
```

Comment:
what does this change stand for?

Reply:
only test: write using batch write 
skip test: write using spark jdbc

because spark jdbc write does not use local time zone, which means it's result is wrong!